src/components/composite/sections/ContextSection.tsx

```diff
@@ -25,9 +25,7 @@ export interface ContextSectionProps {
   impact?: string;
   who?: string;
   references?: Reference[];
-  aspectsLoading?: boolean;
-  canReadAspects?: boolean;
-  canCreateAspects?: boolean;
+  loading: boolean | undefined;
```

Comment:
For such flags I generally use the optional prop signature, `?`. It's similar to `disabled` and `readonly` on inputs for me. It's more like an opinion - free to skip this comment.

Reply:
The criteria here for me sounds as "will this component serve it's main purpose without this prop".
An `entityId` could be `undefined` too, but it can't be omitted if the component is about fetching some data based on the id.